### PR TITLE
chore: Use github secret for read-only token

### PR DESCRIPTION
**Pull request by kbalk**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,19 +12,12 @@ env:
...
-  - TERRAGRUNT_URL=$(echo "$TERRAGRUNT_RELEASES" | jq --raw-output  '.assets[] | select(.name=="terragrunt_linux_amd64") | .browser_download_url')
   - echo "TERRAFORM_URL=${TERRAFORM_URL}"
-  - echo "TERRAGRUNT_URL=${TERRAGRUNT_URL}"
   - curl -sSL -o terraform.zip "$TERRAFORM_URL"
-  - curl -sSL -o terragrunt "$TERRAGRUNT_URL"
   - unzip terraform.zip && rm -f terraform.zip
-  - chmod +x terraform terragrunt
   - mkdir -p $HOME/bin
   - export PATH=$PATH:$HOME/bin
-  - mv terraform terragrunt $HOME/bin
```
Comment (lorengordon) [inline .travis.yml:25]:
```suggestion
  - mv terraform $HOME/bin
```

File: .travis.yml [hunk 2]
```diff
@@ -42,7 +35,6 @@ jobs:
     - stage: test
```
Comment (lorengordon) [inline .travis.yml]: Remove this job entirely, which is this item in the list, with its stage and env.

File: .travis.yml (hunk 1 above)
Comment (kbalk) [reply .travis.yml:25]: ok

File: .travis.yml (hunk 2 above)
Comment (kbalk) [reply .travis.yml]: ok

**Comment by lorengordon** [comment]
Now the failure is just the terraform format. Run `terraform fmt -recursive` with terraform 0.15 and it should be good to go.

**Review by lorengordon — APPROVED** [review]

**Pull request merged**